tests/migrations/test_migrations_order/0001_initial.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'll reduce it to test against `0001.py` and `0002_a.py`. I think that it would be good to keep the suffix test.